gravify/__init__.py

```diff
@@ -38,7 +38,7 @@ def unsecure_url(self):
 
     @property
     def hash(self):
-        return hashlib.md5(self.email.encode("utf-8").lower()).hexdigest()
+        return hashlib.md5(self.email.strip(" ").encode("utf-8").lower()).hexdigest()
```

Comment:
```suggestion
        return hashlib.md5(self.email.strip().encode("utf-8").lower()).hexdigest()
```
I think you should remove this string because without it, any whitespace characters will be removed. With it there however, only spaces are removed.

Reply:
I removed the space identifier from the strip(). I forgot there are tabs and other whitespace characters, good catch...